test/oauth2Auth.spec.ts

```diff
@@ -140,6 +140,37 @@ describe('Oauth2  test', () => {
             oauth2Auth.login('admin', 'admin');
         });
 
+        it('should not hang thee app also if teh logout is missing', function(done)  {
```

Comment:
Misspellings thee  and teh

Reply:
thanks my keyboard is gone... it type e always twice...